### PR TITLE
[Ready to review]Making plot tabs slightly more compact vertically 

### PR DESCRIPTION
**Pull request by coderrsid**
fixing https://github.com/cbioportal/cbioportal/issues/5633

Making plot tabs slightly more compact vertically so that it fits a 13 inch screen easily.
Made it compact, decrease font-size, decrease white spaces and moved the color samples options near the plot.

Changes proposed in this pull request:
- Made it compact
- Decreased font-size
- Removed extra white space
- Moved colour sample options inside plot.

# Screenshots
screenshot 2019-03-08 at 12 16 48 am
screenshot 2019-03-08 at 12 17 05 am

# Notify reviewers
@schultzn @jjgao @alisman

### CODE REVIEW

**Comment by coderrsid** [comment]
@jjgao I made another pull request for this. Please review and merge this soon. It's a request. I want to contribute more.

**Comment by coderrsid** [comment]
Please review @alisman

**Comment by coderrsid** [comment]
Please review @zhx828

**Comment by jjgao** [comment]
Thanks, @coderrsid.

@alisman @adamabeshouse please review.

@adamabeshouse do you think "Show regression line" option should also be moved?

**Comment by coderrsid** [comment]
Please review and merge it soon because i have solved one more issue and i want to make a pull request too.

File: src/pages/resultsView/plots/PlotsTab.tsx [hunk 1]
```diff
@@ -1112,8 +1112,8 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
         }
         return (
             <div>
-                <hr/>
-                <h4>Utilities</h4>
+                {/* <hr/>
+                <h4>Utilities</h4> */}
```
Comment (adamabeshouse) [inline src/pages/resultsView/plots/PlotsTab.tsx]: please remove comment

Comment (coderrsid) [reply src/pages/resultsView/plots/PlotsTab.tsx]: Okay sure.

**Comment by adamabeshouse** [comment]
@coderrsid can you please squash your commits?

**Comment by coderrsid** [comment]
Hey @adamabeshouse Can you please help me. I don't know how to squash commits.

**Comment by alisman** [comment]
@coderrsid you can use git rebase with interactive to squash commits.   just google squashing and you will figure it out.   you should also rebase on latest master.  thanks!

**Comment by coderrsid** [comment]
@alisman I squashed all commits in the last commit. you can see that.

**Comment by alisman** [comment]
@coderrsid I'm still seeing a lot of commits (24) in this PR
image

And I aslso merge commits from master.  There shouldn't be any merge commits from master.   You should include all changes for this ticket in one commit with a descriptive message and then rebase that commit on the current master so that your changes sit on top of the git history in your branch.  Also, you should name your branch something other than master.  Like plotsWidth or something.  

It is sometimes permissable to break up a PR into multiple commits if those commits pertain to sub tasks of a feature that should be separate.  In that case the messages should describe the separate changes.  But a task like you've done here I think really belongs in a single commit.

**Comment by adamabeshouse** [comment]
Hi @coderrsid , you do `git rebase -i HEAD~N` where N is the number of commits you want to squash. It will open a file with the list of all those commits, with "pick" next to all of them. To squash them all into your most recent commit, change all of the words "pick" to "squash", except for the first (topmost) which you should leave as "pick".

This will begin a process where git starts with your first commit, then tries to replay all of the following commits changes onto it, one by one. If there's an issue, it will pause and ask you to fix the issues (like in a normal merge conflict), then once youve resolved it and git add'ed everything, you do `git rebase --continue`. If you think you've messed something up and want to start over, you do `git rebase --abort`.


At the end, you'll be left with one commit which contains all of your changes. Now you need to do what @alisman mentioned: rebase onto master. 

So you have a bit of an issue here because you've made all of your changes on your master branch. Ideally you want to have a separate branch that contains your changes. Here's what you'll need to do.

I'm assuming at this point that you are on your branch `master` which has your single (squashed) commit.

First, move your changes to a new branch: `git checkout -b plots-tab-changes`. Now the branch name `plots-tab-changes` will hold a reference to your commit.

Now, `git checkout master`. This will take you back to master. It's very important you do this before you do the next thing.

Now, you need to see what your name is for cbioportal. Do `git remote -v` to list the names of your remote repositories. One will be cbioportal, one will be yours at coderrsid. If the cbioportal one is called "cbio", then the command will be `git fetch cbio master`. This will get the latest version of our `master` branch at cbioportal, and set the FETCH_HEAD pointer to that branch. Now you'll do `git reset --hard FETCH_HEAD`, which will reset your master branch to our master branch.

Finally, you'll go back to your branch: `git checkout plots-tab-changes`. Now, `git rebase master`. This will replay your changes onto the master branch. Finally, `git push` which will push your `plots-tab-changes` branch to github, where you can make a pull request from it to  cbioportal/master.

Please let me know if you have any questions.

**Comment by jjgao** [comment]
@adamabeshouse that's a good piece of text. Maybe we should have it somewhere in a doc?

**Comment by coderrsid** [comment]
Thanks @adamabeshouse it helped me so much and @jjgao it should be in docs.

**Comment by coderrsid** [comment]
@adamabeshouse @alisman I made a new pull request with all the changes in one commit 
https://github.com/cBioPortal/cbioportal-frontend/pull/2137

**Pull request closed without merging**